discovery-service/src/test/java/org/zowe/apiml/discovery/ApimlInstanceRegistryTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.discovery;
+
+import com.netflix.appinfo.*;
+import com.netflix.discovery.DiscoveryClient;
+import com.netflix.discovery.EurekaClient;
+import com.netflix.discovery.EurekaClientConfig;
+import com.netflix.eureka.*;
+import com.netflix.eureka.resources.ServerCodecs;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.mockito.Mockito.*;
+import org.junit.jupiter.api.Nested;
+
+import org.springframework.cloud.netflix.eureka.server.InstanceRegistryProperties;
+import org.springframework.context.ApplicationContext;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ApimlInstanceRegistryTest {
```

Comment:
would be good to have some integration test for this, to see if it really works as it should

Reply:
Good point, I will add it